projects/js-packages/components/components/threats-data-view/styles.module.scss

```diff
@@ -5,9 +5,13 @@
 		min-height: 0;
 	}
 
-    .dataviews-view-table td, .dataviews-view-table th {
-        white-space: initial;
-    }
+	.dataviews-view-table td, .dataviews-view-table th {
+		white-space: initial;
+
+		svg {
+			margin: 0;
+		}
```

Comment:
This style is going to apply to all SVGs inside any dataview table cell. This is especially relevant as we are inside the `:global` context so these styles will affect the entire page they are loaded into.

We should use a more specific way to target the specific element/case that this is for, i.e. give the `<Spinner>` a unique class name to target.